nuget/net462/NUnit3TestAdapter.targets

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Handle the coexistance between Microsoft.Testing.Platform and Microsoft.NET.Test.Sdk  -->
+  <PropertyGroup>
+    <GenerateTestingPlatformEntryPoint Condition=" '$(GenerateTestingPlatformEntryPoint)' == '' ">$(EnableNUnitRunner)</GenerateTestingPlatformEntryPoint>
+    <GenerateProgramFile Condition=" '$(EnableNUnitRunner)' == 'true' ">false</GenerateProgramFile>
```

Comment:
The first line enables/disables the generation of the entry point for the new platform and the second line disables the VSTest entry point if the new platform is enabled.

Reply:
Is it necessary to disable the VSTest entry point?  
It would be better if the user decides without having to use any parameters.  If he runs it using `dotnet test`, it would use that entry point, if he runs the executable, it would use that entry point.

Reply:
VSTest always creates its own entry point. Our new platform also wants to auto-generate it (to simplify user experience) so we need to have them mutually exclusive.

> If he runs it using dotnet test

Not really, as you could see in the example because `dotnet test` is pretty common gesture, we have a mode to hook into it skipping VSTest redirection and calling the runner instead.